rust/src/smb/events.rs

```diff
@@ -30,6 +30,21 @@ pub enum SMBEvent {
     NegotiateMalformedDialects = 6,
 }
 
+impl SMBEvent {
```

Comment:
Looks like this should have been in its own commit?

Reply:
Will move to its own commit.